components/CountdownContainer.js

```diff
@@ -26,10 +26,25 @@ class CountdownContainer extends React.Component {
 function renderCompleted () {
   return (
     <span>
-      Good luck and have fun.
+      Remaining ticket released!
       <style jsx>{`
         span {
           font-size: 36px;
+          position: relative;
+          bottom: 10px;
+          opacity: 0; 
+          animation: fade-in-from-top 1.5s;
+          animation-fill-mode: forwards;
+        }
+        @keyframes fade-in-from-top {
+          from {
+            opacity: 0;
+            bottom: 20px;
```

Comment:
Thank you for your feedback. I've changed the property `bottom` to `transform` instead.